src/Button.js

```diff
@@ -0,0 +1,6 @@
+import React from "react"
+const Button = () => {
+return (<button className="btn btn-primary">Search IDs</button>)
```

Comment:
How could we make this component more reusable? Right now if we tried to use this component elsewhere it would always say "Search IDs". What could we do to make it more useful in other parts of our app?